packages/reactstrap-validation-select/src/AvSelect.js

```diff
@@ -259,6 +261,17 @@ class AvSelect extends AvBaseInput {
       Tag = Creatable;
     }
 
+     // Enhance placeholder for accessibility
+  placeholder = (
+    <>
+      {placeholder || 'Select...'}
+      <span className="sr-only">
+        {(touched && hasError && ariaFeedback?.errorMessage) || null } 
+        {helpMessage || null}
```

Comment:
We should fix this linter error. Seems like `helpMessage` is not defined in this component